ui/lesson/ScriptingChallenge/Runner/index.tsx

```diff
@@ -129,23 +129,15 @@ export default function Runner({
           }
           case 'debug': {
             // payload = payload.trim()
-            // terminal('print', payload)
+            // sendTerminal('print', payload)
             break
           }
           case 'output': {
             payload = payload.trim()
             sendTerminal('print', payload)
 
             const [res, err] = await onValidate(payload)
-            if (!res) {
-              setHasherState(HasherState.Error)
-              setIsRunning(false)
-              setState(State.Complete)
-              sendTerminal('error', err)
-              ws?.close()
-              return
-            }
-
+            if (!res) break
```

Comment:
I think it is here where the break occurs instead of returning the error, I guess this is what you had discussed with removing some of the challenge specific errors?